hoverctl/wrapper/hoverfly.go

```diff
@@ -188,8 +189,45 @@ func SetMiddleware(target Target, binary, script, remote string) (v2.MiddlewareV
 	return apiResponse, nil
 }
 
+func GetLogs(target Target, format string) ([]string, error) {
+	headers := map[string]string{
+		"Content-Type": "text/plain",
+	}
+
+	if format == "json" {
+		headers["Content-Type"] = "application/json"
+	}
+
+	response, err := doRequest(target, "GET", v2ApiLogs, "", headers)
+	if err != nil {
+		return []string{}, err
+	}
+
+	defer response.Body.Close()
+
+	responseBody, _ := ioutil.ReadAll(response.Body)
```

Comment:
No, I am working with the JSON as a string and trimming off the bits I don't need to print.

The logs endpoint will return a JSON object that looks like
```
{
    "logs": [
         {
             "level": "info",
              "msg": "something"
         },
         {
             "level": "info",
              "msg": "something"
         }
    ]
}
```

Previously, `hoverctl logs` would print logs such as
```
{"level":"info","method":"GET","msg":"started handling request","remote":"[::1]:61802","request":"/api/v2/logs","time":"2017-05-16T11:26:22+01:00"}
```
So I am trimming `{"logs":[` from the start and `]}` from the end. I am also removing the `,` from the end of each log.

I could have unmarshalled  them into structs and then iterated through each of the individual log elements and marshalled them back into JSON. But I think what have done is simpler. I can change it to unmarshal and then marshal again.